src/View/Sidebar.vala

```diff
@@ -223,15 +223,15 @@ namespace Marlin.Places {
                                 "active", Column.SHOW_SPINNER,
                                 "pulse", Column.SPINNER_PULSE);
 
-            crpb = new Gtk.CellRendererPixbuf (); /* Icon for eject button (hidden while ejecting or unmounted) */
+            crpb = new Gtk.CellRendererPixbuf (); /* Icon for eject button  (hidden while ejecting or unmounted) and another signs */
```

Comment:
Double space before bracket in comment. "and another signs" should be "or other icons". Overlong line.  Could probably lose the words in brackets.